src/components/TopBar/Controls.js

```diff
@@ -31,17 +31,17 @@ const controlsInjector = inject(({ store }) => {
 
 export const Controls = controlsInjector(observer(({ store, history, annotation }) => {
   const isReview = store.hasInterface('review');
-  
+
   const historySelected = isDefined(store.annotationStore.selectedHistory);
-  const { userGenerate, sentUserGenerate, versions, results, editable: annotationEditable } = annotation;
+  const { userGenerate, sentUserGenerate, versions, results, editable } = annotation;
   const buttons = [];
 
   const [isInProgress, setIsInProgress] = useState(false);
 
   // const isReady = store.annotationStore.selected.objects.every(object => object.isReady === undefined || object.isReady);
-  const disabled = !annotationEditable || store.isSubmitting || historySelected || isInProgress; // || !isReady;
+  const disabled = !editable || store.isSubmitting || historySelected || isInProgress; // || !isReady;
```

Comment:
Should this not also be observing `annotation.isReadonly()` instead of `annotation.editable`?

Reply:
No, we never locked the submit/update buttons in readonly. I would leave it like this until we decide something else.